docs/guide/writing-tasks.md

```diff
@@ -876,6 +876,10 @@ Additional container can be very handy in many scenarios. Please check [Cirrus C
 ??? tip "Port Mapping"
     It's also possible to map ports of additional containers by using `<HOST_PORT>:<CONTAINER_PORT>` format for the `port` field.
     For example, `port: 80:8080` will map port `8080` of the container to be available on local port `80` within a task.
+  
+    **Note:** don't use port mapping unless absolutely necessary. For example, when you have several additional containers
+    which start the service on the same port and there's no easy way to change that. Port mapping limits 
```

Comment:
Grammar - this sentence ends too abruptly

Reply:
Which one? "Port mapping limits"? There is continuation on the next line. 🤔

Reply:
`that. Port mapping limits` You are giving an example here, and then end the sentence.  You don't explain what this example is for until the next sentence, which makes very little sense,

Reply:
See https://github.com/cirruslabs/cirrus-ci-docs/pull/811/files#r595227933 for a fix.